lib/beaker/testmode_switcher/runner_base.rb

```diff
@@ -0,0 +1,14 @@
+module Beaker
+  module TestmodeSwitcher
+    # Reused funtionality for all the runners
+    class RunnerBase
+      def get_acceptable_puppet_run_exit_codes(opts = {})
+        return [0] if opts[:catch_changes]
+        return [0, 2] if opts[:catch_failures]
+        return [1, 4, 6] if opts[:expect_failures]
+        return [2] if opts[:expect_changes]
+        [0...256]
```

Comment:
This is not the same return as was previously there. `[0...256]` is an array with a single element, `(0...256)` is just the Range.